org.eclipse.winery.frontends/app/tosca-management/src/app/wineryLicenseModule/wineryLicense.component.html

```diff
@@ -11,50 +11,174 @@
+                                    *ngIf="isFailed()">
+                                Reset
+                            </button>
+                        </div>
+                    </form>
+                </mat-step>
+                <mat-step completed="false">
+                    <form [formGroup]="secondFormGroup">
+                        <ng-template matStepLabel>Check Compatibility</ng-template>
+                        <br>
+                        <mat-label style="font-size: 15px">{{foundLicenses}}</mat-label>
+                        <br><br>
+                        <div>
+                            <button class="btn btn-primary" matStepperPrevious>Back</button>
+                            <button style="margin-left: 12px" class="btn btn-primary"
+                                    (click)="onSubmitCheckCompatibility()">Next
+                            </button>
+                        </div>
+                    </form>
+                </mat-step>
+                <mat-step>
+                    <ng-template matStepLabel>Select License</ng-template>
+                    <div>
+                        <mat-action-list>
+                            <mat-list-item *ngFor="let compatibleLicense of compatibleLicenses">
+                                <button mat-list-item
+                                        (click)="selectLicense(compatibleLicense)"> {{compatibleLicense}} </button>
+                            </mat-list-item>
+                        </mat-action-list>
+                        <button class="btn btn-primary" (click)="stepper.reset()">Reset</button>
+                    </div>
+                </mat-step>
+            </mat-horizontal-stepper>
+        </div>
+
+        <ng-template #confirmSaveModal>
+            <winery-modal-header [modalRef]="confirmSaveModalRef"
+                                 [title]="'Are you sure you want to save the license text without modifying it?'">
+            </winery-modal-header>
+            <winery-modal-body>
+                <p>
+                    Depending on the selected license, some fields like the date or the copyright holders have to be
+                    adjusted!
+                </p>
+            </winery-modal-body>
+            <winery-modal-footer (onOk)="saveLicenseFile();" [modalRef]="confirmSaveModalRef"
+                                 [closeButtonLabel]="'No (Edit)'"
+                                 [okButtonLabel]="'Yes (Save)'">
+            </winery-modal-footer>
+        </ng-template>
+
+        <ng-template #confirmDownloadModal>
+            <winery-modal-header [modalRef]="confirmDownloadModalRef"
+                                 [title]="'Do you want to download the license file?'">
+            </winery-modal-header>
+            <winery-modal-body>
+                <p>
+                    By saving, the license text was stored in the ROAR.
```

Comment:
I guess you should omit the `By saving` part.

Reply:
changed